.github/workflows/osx.yml

```diff
@@ -28,5 +28,8 @@ jobs:
         cmake .. -DBUILD_SHARED_LIBS=${{ matrix.config.shared }} -DCMAKE_BUILD_TYPE=${{ matrix.config.build_type }}
         make -j8 
         ctest -j10
+    - name: test_install
```

Comment:
You name this step `install` in the Windows step. Either are good to me, but maybe keep only one for uniformity.